src/pybind/mgr/orchestrator/module.py

```diff
@@ -2361,7 +2361,10 @@ def _upgrade_status(self) -> HandleCommandResult:
             'is_paused': status.is_paused,
         }
         out = json.dumps(r, indent=4)
-        return HandleCommandResult(stdout=out)
+        if r.get('in_progress'):
+            return HandleCommandResult(stdout=out)
+        else:
+            return HandleCommandResult(stdout="There are no upgrades in progress currently.")
```

Comment:
```suggestion
        if r.get('in_progress'):
            return HandleCommandResult(stdout=out)
        return HandleCommandResult(stdout="There are no upgrades in progress currently.")
```

Reply:
Hi, This should simply be a message displayed on the CLI if no upgrades are available.